src/test/java/com/tuplejump/stargate/cassandra/DropAndRecreateTest.java

```diff
@@ -36,10 +42,16 @@ public void shouldAllowToRecreateIndex() throws InterruptedException {
         createKS(keyspace);
         createTableAndIndex(false);
         try {
+            List<String> fetchedKeys = new LinkedList<String>();
+            Assert.assertTrue(assertResult(getResults("TAG", "", false),
+                    Arrays.asList("8", "7", "5", "6", "2", "3", "1", "4"), "key"));
             Assert.assertEquals(8, countResults("TAG", "", false, false));
             Assert.assertEquals(3, countResults("TAG", "category = '" + mq("state", "CA") + "'", true));
             getSession().execute("DROP INDEX dropcreate;");
             createTableAndIndex(true);
+
+            Assert.assertTrue(assertResult(getResults("TAG", "", false),
```

Comment:
assertTrue(assertResult) sounds wrong. You can do the assertion in the assertResults itself.